control_panel_api/migrations/0002_auto_20170816_1525.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.4 on 2017-08-16 15:25
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django_extensions.db.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('control_panel_api', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='App',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created', django_extensions.db.fields.CreationDateTimeField(auto_now_add=True)),
+                ('name', models.CharField(max_length=100)),
+                ('slug', django_extensions.db.fields.AutoSlugField(blank=True, editable=False, populate_from='name')),
+                ('repo_url', models.CharField(blank=True, default='', max_length=255)),
```

Comment:
It may be more than plenty but a part of me wonder if limit this URL to `255` chars may be a problem for some funny named repos.